dashboard/test/load/database/README.md

```diff
@@ -0,0 +1,33 @@
+# Using Sysbench for Database Load Testing
+
+[sysbench](https://github.com/akopytov/sysbench) is a scriptable multi-threaded benchmark tool based on LuaJIT. It is most frequently used for database benchmarks, but can also be used to create arbitrarily complex workloads that do not involve a database server.  AWS has published [a white paper](https://d1.awsstatic.com/product-marketing/Aurora/RDS_Aurora_Performance_Assessment_Benchmarking_v1-2.pdf) outlining steps to assess the performance of an RDS Aurora database.
+
+## Setup
+
+Here are steps to using sysbench to carry out benchmark testing of the Code.org Aurora database:
+
+1. **Create Stack** - Provision a load test CloudFormation Stack using dashboard/test/load/database/aurora-loadtest.yml setting the DBClusterIdentifier Parameter to the identifier that you plan to use for the load test database cluster that you will provision in the next step. Beware that AWS automatically adds `-cluster` to the end of the identifier you specify when creating the Database cluster in step 2, so include that suffix here.
+1. **Create Database Cluster**
+    1. Restore a recent Snapshot of the production Aurora cluster into the VPC created by the load test Stack, configuring it to use the VPC, DB Subnet Group and MySQL Security Group created by the load test Stack and using the production Cluster Parameter Group and Writer Instance Parameter Group. Enable Performance Insights on the cluster and enable all log types (Slow Query, Error, General, etc.).
+    1. After the restore is complete, change the password of the admin SQL login via the AWS web console using the Modify Database Instance page.
```

Comment:
It would be great to automate this step- extra great if it could be automated by leveraging some of our existing CloudFormation template code (specifically the `database.yml.erb` component), to ensure that tests are running on resources configured similarly to our standard infrastructure.

I think this would probably involve playing around with the [`RestoreType`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-dbcluster.html#cfn-rds-dbcluster-restoretype) property of the `AWS::RDS::DBCluster` resource.

Reply:
We previously were not able to clone our production database cluster into a non-default DB Subnet using CloudFormation RestoreToPointInTime, but a fix is finally on the way https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-rds/pull/55